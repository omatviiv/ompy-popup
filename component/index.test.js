import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Component, { thm } from 'componnt';

describe('Component', () => {
  it('renders with value', () => {
    const wrapper = shallow(<Component value='test' />);
    expect(wrapper).toMatchSnapshot();
  });
  describe('theme', () => {
    it('Main', () => {
      const wrapper = renderer.create(<thm.Main />).toJSON();
      expect(wrapper).toMatchSnapshot();
    });
    it('Strong', () => {
      const wrapper = renderer.create(<thm.Strong />).toJSON();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
