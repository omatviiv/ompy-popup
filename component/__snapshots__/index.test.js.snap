// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders with value 1`] = `
<styled.div
  value="test"
  visible={false}
/>
`;

exports[`Component theme Main 1`] = `
.c0 {
  display: none;
  padding: 5px 10px;
  background: white;
  border: 1px solid grey;
  position: absolute;
}

<div
  className="c0"
/>
`;
